Makefile

```diff
@@ -1,10 +1,121 @@
-# Edit setup.py and increment version number
+.PHONY: clean data lint format requirements
 
-all:
-	pandoc -s README.md -o README.rst
+#################################################################################
+# GLOBALS                                                                       #
+#################################################################################
+
+PROJECT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+PROJECT_NAME = empiricaldist
+PYTHON_VERSION = 3.10
+PYTHON_INTERPRETER = python
+
+
+#################################################################################
+# COMMANDS                                                                      #
+#################################################################################
+
+
+## Set up python interpreter environment
+create_environment:	
+	conda create --name $(PROJECT_NAME) python=$(PYTHON_VERSION) -y
+	@echo ">>> conda env created. Activate with:\nconda activate $(PROJECT_NAME)"
+
+
+## Install Python Dependencies
+requirements:
+	$(PYTHON_INTERPRETER) -m pip install -U pip setuptools wheel
+	$(PYTHON_INTERPRETER) -m pip install -r requirements.txt 
+	$(PYTHON_INTERPRETER) -m pip install -r requirements-dev.txt 
+	
+
+## Delete all compiled Python files
+clean:
+	find . -type f -name "*.py[co]" -delete
+	find . -type d -name "__pycache__" -delete
+
+
+## Format source code with black
+format:
+	black --config pyproject.toml empiricaldist
+
+
+## Lint using flake8 and black (use `make format` to do formatting)
+lint:
+	flake8 empiricaldist
+	black --check --config pyproject.toml empiricaldist
+
+
+tests:
+	pytest --no-cov empiricaldist
```

Comment:
Looks like `--cov` is set in the opts, but `--no-cov` is set here. I'd just set this in one place